command/pki_verify_sign_command.go

```diff
@@ -0,0 +1,231 @@
+package command
+
+import (
+	"bytes"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/hashicorp/vault/api"
+	"github.com/hashicorp/vault/sdk/helper/certutil"
+	"github.com/ryanuber/columnize"
+)
+
+type PKIVerifySignCommand struct {
+	*BaseCommand
+
+	flagConfig          string
+	flagReturnIndicator string
+	flagDefaultDisabled bool
+	flagList            bool
+}
+
+func (c *PKIVerifySignCommand) Synopsis() string {
+	return "Check Whether One Certificate Validates Another Specified Certificate"
+}
+
+func (c *PKIVerifySignCommand) Help() string {
+	helpText := `
+Usage: vault pki verify-sign POSSIBLE-ISSUER POSSIBLE-ISSUED
+Here POSSIBLE-ISSUER and POSSIBLE-ISSUED are the fully name-spaced path to the certificate,
+for instance: 'ns1/mount1/issuer/issuerName/json'
+Returns five fields of information:
+- signature_match: was the key of the issuer used to sign the issued
+- path_match: the possible issuer appears in the valid certificate chain of the issued
+- key_id_match: does the key-id of the issuer match the key_id of the subject
+- subject_match: does the subject name of the issuer match the issuer subject of the issued
+- trust_match: if someone trusted the parent issuer, is the chain provided sufficient to trust the child issued
+`
+	return strings.TrimSpace(helpText)
+}
+
+func (c *PKIVerifySignCommand) Flags() *FlagSets {
+	set := c.flagSet(FlagSetHTTP | FlagSetOutputFormat)
+	return set
+}
+
+func (c *PKIVerifySignCommand) Run(args []string) int {
+	f := c.Flags()
+	if err := f.Parse(args); err != nil {
+		c.UI.Error(err.Error())
+		return 1
+	}
+
+	args = f.Args()
+
+	if len(args) < 2 {
+		if len(args) == 0 {
+			c.UI.Error("Not enough arguments (expected potential issuer and issued, got nothing)")
+		} else {
+			c.UI.Error("Not enough arguments (expected both potential issuer and issued, got only one)")
+		}
+		return 1
+	} else if len(args) > 2 {
+		c.UI.Error(fmt.Sprintf("Too many arguments (expected only potential issuer and issued, got %d arguments)", len(args)))
+		for _, arg := range args {
+			if strings.HasPrefix(arg, "-") {
+				c.UI.Warn(fmt.Sprintf("Options (%v) must be specified before positional arguments (%v)", arg, args[0]))
+				break
+			}
+		}
+		return 1
+	}
+
+	issuer := sanitizePath(args[0])
+	issued := sanitizePath(args[1])
+
+	client, err := c.Client()
+	if err != nil {
+		c.UI.Error(fmt.Sprintf("Failed to obtain client: %s", err))
+		return 1
+	}
+
+	err, results := verifySignBetween(client, issuer, issued)
+	if err != nil {
+		c.UI.Error(fmt.Sprintf("Failed to run verification: %v", err))
+		return pkiRetUsage
+	}
+
+	c.outputResults(results, issuer, issued)
+
+	return 0
+}
+
+func verifySignBetween(client *api.Client, issuerPath string, issuedPath string) (error, map[string]bool) {
```

Comment:
A nit for a future PR: Go return arg order is typically reversed, with the `error`/ok boolean being the last item.